aten/src/ATen/native/quantized/cpu/qfc.cpp

```diff
@@ -0,0 +1,152 @@
+#include <ATen/ATen.h>
+#include <ATen/core/op_registration/op_registration.h>
+#include <ATen/cpp_custom_type_hack.h>
+#include <ATen/fbgemm_utils.h>
+
+#include <algorithm>
+#include <tuple>
+
+namespace at {
+namespace native {
+namespace {
+
+class QFCInt8 final : public c10::OperatorKernel {
+ public:
+#ifdef USE_FBGEMM
+  std::tuple<at::Tensor, double, int64_t> operator()(
+      const at::Tensor& input,
+      double input_scale,
+      int64_t input_zero_point,
```

Comment:
A similar question here. int32_t or int should be good enough here.
nit: if we want to strictly follow the standard, int64_t should be std::int64_t

Reply:
Same here. Please ignore.